samples/ImplicitFlow01/AppWithFetchClient/Controllers/AuthorizationController.cs

```diff
@@ -0,0 +1,125 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Text;
+using System.Threading.Tasks;
+using AspNet.Security.OpenIdConnect.Extensions;
+using AspNet.Security.OpenIdConnect.Primitives;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Options;
+using OpenIddict.Abstractions;
+using OpenIddict.Core;
+using OpenIddict.EntityFrameworkCore.Models;
+using OpenIddict.Server;
+
+public class AuthorizationController : Controller
+{
+    private readonly IOptions<IdentityOptions> _identityOptions;
+    private readonly OpenIddictScopeManager<OpenIddictScope> _scopeManager;
+    private readonly SignInManager<IdentityUser> _signInManager;
+    private readonly UserManager<IdentityUser> _userManager;
+
+    public AuthorizationController(
+        IOptions<IdentityOptions> identityOptions,
+        OpenIddictScopeManager<OpenIddictScope> scopeManager,
+        SignInManager<IdentityUser> signInManager,
+        UserManager<IdentityUser> userManager)
+    {
+        _identityOptions = identityOptions;
+        _scopeManager = scopeManager;
+        _signInManager = signInManager;
+        _userManager = userManager;
+    }
+
+    [HttpGet("~/connect/authorize")]
+    public async Task<IActionResult> Authorize(OpenIdConnectRequest oidcRequest)
```

Comment:
nit: you may want to make the parameter name consistent with the next method and the other samples.

Reply:
That's a good idea. I made the change because to resolve a model binding warning. `OpenIdConnectRequest` has a `Request` property that apparently might conflict with a method parameter named `request`.

Reply:
Done (thought GitHub doesn't register that.)